packages/@vuepress/markdown/__tests__/__snapshots__/snippet.spec.js.snap

```diff
@@ -10,17 +10,11 @@ exports[`snippet import snippet 1`] = `
 exports[`snippet import snippet with highlight multiple lines 1`] = `
 <div class="highlight-lines">
   <div class="highlighted">&nbsp;</div>
-  <div class="highlighted">&nbsp;</div>
-  <div class="highlighted">&nbsp;</div>
-  <br>
-</div>export default function () { // .. }
+</div>
 `;
```

Comment:
@Shigma Note that this snap shouldn't be removed, I guess this case need to be updated.

Reply:
Fixed at #1398